src/core/properties/cameraproperty.cpp

```diff
@@ -56,16 +56,22 @@ CameraProperty::CameraProperty(const std::string& identifier, const std::string&
     , cameraType_("cameraType", "Camera Type",
                   InviwoApplication::getPtr()->getCameraFactory()->getKeys(), 0)
     , cameraActions_("actions", "Actions", buttons())
-    , lookFrom_("lookFrom", "Look from", eye, -vec3(100.0f), vec3(100.0f), vec3(0.1f),
+    , lookFrom_("lookFrom", "Look from", eye, {-vec3(100.0f), ConstraintBehaviour::Ignore},
```

Comment:
I would have made this change in a separate PR (this is a breaking change, while the other ones are not)

Reply:
yes